applications/GeoMechanicsApplication/tests/test_GeoMechanicsApplication.py

```diff
@@ -78,7 +78,6 @@ def AssembleTestSuites():
                         KratosGeoMechanicsChangingWaterLevelTests,
                         KratosGeoMechanicsStrainMeasureTests,
                         KratosGeoMechanicsSetMultipleMovingLoadProcessTests,
```

Comment:
Sorry I do not know why but a comma is not used in the last lines. Perhaps, it is good to remove the comma in this line as well.

Reply:
Python accepts it without problems, thats why I did not notice it remained. Removed it.